### PR TITLE
fix that -> this

### PR DESCRIPTION
**Pull request by jonyhy96**
fix that -> this

```release-note
None
```

### CODE REVIEW

File: docs/concepts/overview/what-is-kubernetes.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ This page is an overview of Kubernetes.
 
 {% capture body %}
 Kubernetes is a portable, extensible open-source platform for managing
-containerized workloads and services, that facilitates both
+containerized workloads and services, this facilitates both
```
Comment (zacharysarah) [inline docs/concepts/overview/what-is-kubernetes.md]: Should instead be:

> containerized workloads and services that facilitates both

Comment (jonyhy96) [reply docs/concepts/overview/what-is-kubernetes.md]: thanks for your reply
i think i was wrong but the correct format should be :
containerized workloads and services that facilitate both

**Comment by jonyhy96** [comment]
the commit doesn't make sense
close it

**Pull request closed without merging**